src/utils/questions.js

```diff
@@ -0,0 +1,105 @@
+const getManagerQuestions = () => {
+  const managerQuestions = [
```

Comment:
We don't need to declare a variable if the only thing we are doing is returning it. We can simply return this array directly: 

``` 
return [ ... questions here ]
```